config.toml

```diff
@@ -127,6 +127,9 @@ nextUrl = "https://kubernetes-io-vnext-staging.netlify.com/"
 githubWebsiteRepo = "github.com/kubernetes/website"
 githubWebsiteRaw = "raw.githubusercontent.com/kubernetes/website"
 
+# GitHub repository links for editing a page and opening issues.
+github_repo = "https://github.com/kubernetes/website"
```

Comment:
This is similar to `githubWebsiteRepo`; I'd either merge these or use consistent capitalization (eg: change `githubWebsiteRepo` to `github_repo_short_name`)

Maybe something to log as an issue for some follow-up work?

Reply:
Yes, I will remove the other variables if there are no dependencies.

Reply:
This variable is used by the theme.

Reply:
Future tidying: we could use the new `github_repo` parameter in `layouts/partials/git-info.html`, to generate the links to page history.

Reply:
Instead, I added in the theme's git-info partial, "page-meta-lastmod.html" and updated the
date format in config.toml.